index.js

```diff
@@ -133,8 +133,8 @@ function searchES(es, submittedQuery, callback) {
       must.push({ "range": { "created_at": { "lte": dateRange[1]}}})
     }
     */
-    queryTerm.filtered = {
-      query: textQuery,
+    queryTerm.bool = {
+      must: textQuery,
```

Comment:
this was the big syntax change to figure out 💡

Reply:
dev blog entry here https://github.com/enjalot/blockbuilder-search/issues/31#issuecomment-415992968